apps/builder/app/builder/features/style-panel/shared/model.tsx

```diff
@@ -221,7 +222,7 @@ export const getDefinedStyles = ({
 
 export const $definedStyles = computed(
   [
-    $selectedInstancePathWithRoot,
+    $selectedInstancePath,
```

Comment:
are you sure nothing will break?

Reply:
no, but I tried o find out, didn't seem like broken, pls check as well